required/firstaid/config-pdf.lua

```diff
@@ -0,0 +1,6 @@
+-- Tests needing \pdfoutput=1
+
+stdengine    = "pdftex"
```

Comment:
Up-to-date `l3build` doesn't need `strengine` as it uses the first entry in `checkengines` (it does no harm, just for future ref.)